aries-site/src/examples/templates/filtering/FilteringCards2/FilteringCards2.js

```diff
@@ -0,0 +1,158 @@
+import { useContext, useEffect } from 'react';
+import styled from 'styled-components';
+import {
+  Box,
+  Card,
+  CardBody,
+  Grid,
+  Heading,
+  ResponsiveContext,
+  Text,
+} from 'grommet';
+
+import { FilterControls, FilterContext } from '../../FilterControls';
+import { users } from './mockData';
+
+const StyledCard = styled(Card)`
+  will-change: transform;
+  transition: all 0.15s ease-in-out;
+  :focus,
+  :hover {
+    transform: scale(1.01, 1.01);
+  }
+`;
+
+export const FilteringCards2 = () => {
+  const filterContext = useContext(FilterContext)();
+
+  return (
+    <>
+      <Heading level={2} margin={{ bottom: 'small', top: 'none' }}>
+        Users
+      </Heading>
+      <FilterContext.Provider value={{ ...filterContext, data: users }}>
+        {/* FilterControls provides filtering mechanisms to affect our 
+        result set, in this case, Users. */}
+        <FilterControls />
+        <Users />
+      </FilterContext.Provider>
+    </>
+  );
+};
+
+const Users = () => {
+  const size = useContext(ResponsiveContext);
+  const {
+    data,
+    filteredResults,
+    setFilteredResults,
+    setFilterAttributes,
+    filters,
+    filtersLayer,
+    getFilteredResults,
+    searchValue,
+  } = useContext(FilterContext);
+
+  /* Configure which attributes should be made available for the user 
+     to filter upon */
+  useEffect(() => {
+    const attributes = [
+      { property: 'role', header: 'Role', filterType: 'checkboxgroup' },
```

Comment:
Any reason to have the component all lowercase as opposed to camel case or matching the component name "CheckBoxGroup"?

Reply:
I'll change to matching component name.